python/src/aiconfig/editor/client/src/components/ConfigNameDescription.tsx

```diff
@@ -80,11 +80,16 @@ export default memo(function ConfigNameDescription({
       if (readOnly) {
         return;
       }
-      setIsEditing(true);
-      if (event.target === nameDisplayRef.current) {
+
+      if (event.currentTarget === nameDisplayRef.current) {
         setInitialFocus("name");
-      } else if (event.target === descriptionDisplayRef.current) {
+        setIsEditing(true);
+      } else if (
+        event.currentTarget === descriptionDisplayRef.current &&
+        !isClickableChildElement(event.target)
+      ) {
         setInitialFocus("description");
+        setIsEditing(true);
```

Comment:
Why do we re-set this to true when hitting enter? Don't we already exit out on L72?